RELEASE.md

```diff
@@ -1,3 +1,33 @@
+## Release 0.0.16
+
+## New features
+- Bulk search (https://github.com/marqo-ai/marqo/pull/363, https://github.com/marqo-ai/marqo/pull/373). 
+Conduct multiple searches with just one request. This improves search throughput in Marqo by parallelising multiple search queries in a single API call. 
+The average search time can be decreased up to 30%, depending on your devices and models. 
+Check out the usage guide [here](https://docs.marqo.ai/0.0.16/API-Reference/bulk)
+- Configurable number of index replicas (https://github.com/marqo-ai/marqo/pull/391). 
+You can now configure how many replicas to make for an index in Marqo using the `number_of_replicas` parameter. Marqo makes 1 replica by default.
+We recommend having at least one replica to prevent data loss. Replicas also improve search performance, so you might want to add more if you have a read-heavy workload. 
```

Comment:
Remove this: `Replicas also improve search performance, so you might want to add more if you have a read-heavy workload. `

Reply:
done